continue-reading.html

```diff
@@ -2,12 +2,151 @@
 <html>
   <head>
     <title>Continue Reading</title>
+    <style>
+      @import "index.css";
+    </style>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
   </head>
   <body>
-    <section id="header"></section>
-    <section id="main"></section>
-    <section id="footer"></section>
+
+    <section id="header2">
+      <h1><a class="header-link" href="index.html">CONTINUE READING PAGE</a></h1>
+      <H6 class=>A PAGE FOR READING CONTENT</H6>
+    </section>
+
+    <section id="main2"> 

```

Comment:
try to treat your id names and your class names similarly to what you would use for variables in you JavaScript